Sources/ArcGISToolkit/Common/AttachmentPreview.swift

```diff
@@ -193,6 +223,17 @@ struct AttachmentPreview: View {
             .quickLookPreview($url)
         }
     }
+    
+    /// Updates `cellSize` based on the provided container width, `cellBaseWidth`,
+    /// `cellSpacing`, and `visiblePortion` such that the `cellVisiblePortion` width of
+    /// one cell is shown to indicate scrollability.
+    /// - Parameter width: The width of the container the `AttachmentPreview` is in.
+    func updateCellSizeForContainer(_ width: CGFloat) {
+        let fullyVisible = modf(width / cellBaseWidth)
+        let totalPadding = fullyVisible.0 * cellSpacing
+        let newWidth = (width - totalPadding) / (fullyVisible.0 + cellVisiblePortion)
+        cellSize = CGSize(width: newWidth, height: newWidth)
```

Comment:
> 1. The original code had a condition to only compute this if `!editControlsDisabled`. I didn't see a reason to keep that condition around. @mhdostal do you recall why it was in there?

That was strictly so the Popup display didn't change. I'm OK with making the change here.

> 2. Because the `GeometryReader` only reads the width provided to `AttachmentPreview`, we didn't need to factor in leading padding since there is none on the first cell.
👍